frontend/discourse/app/lib/lightbox.js

```diff
@@ -240,6 +240,36 @@ export default async function lightbox(elem, siteSettings) {
       return data;
     });
 
+    const itemsToPreload = items.filter((item) => {
+      const { largeSrc, targetWidth, targetHeight } = item.dataset;
+      const hasImageSrc = largeSrc || item.getAttribute("href");
+      const missingDimensions = !targetWidth || !targetHeight;
+      const imgDimensions = item
+        .querySelector(".informations")
+        ?.textContent.split(" ")[0];
+      const missingMetaData = !imgDimensions
+        ?.split(/x|×/)
+        .every((d) => !!d.trim());
```

Comment:
```suggestion
        ?.textContent.trim()
        .split(" ")[0];
      const missingMetaData = !imgDimensions?.split(/x|×/).every((d) => !!d);
```

I've tested this locally and there is still a small issue where the data attributes get applied unnecessarily when the image dimensions exist in the informations div. I moved the trim position to before the string is split and it seems to work better now and covers the case of blank values too.

Reply:
Thank you, it works now.
I’ll commit the suggestion.